lib/kafka_ex.ex

```diff
@@ -299,7 +299,7 @@ defmodule KafkaEx do
     offset_commit_api_version = Keyword.get(opts, :offset_commit_api_version, 0)
 
     retrieved_offset =
-      current_offset(supplied_offset, partition, topic, worker_name)
+      current_offset(supplied_offset, partition, topic, worker_name, offset_fetch_api_version)
```

Comment:
I might not be looking straight, but I don't think `offset_fetch_api_version` is defined in this function ?

(also we'll have to update the function doc)

Reply:
Ok ok, thank, it's updated